src/main/java/com/treasuredata/client/TDRequestErrorHandler.java

```diff
@@ -60,6 +60,43 @@ protected SimpleDateFormat initialValue()
         }
     };
 
+    /**
+     * Show or suppress warning messages for TDClientHttpException
+     */
+    private static TDClientHttpException clientError(TDClientHttpException e, ResponseContext responseContext)
+    {
+        boolean showWarning = true;
+        boolean showStackTrace = false;
+        int code = e.getStatusCode();
+        switch (code) {
+            case HttpStatus.NOT_FOUND_404:
+                if (responseContext.apiRequest.getPath().startsWith("/v3/table/distribution")) {
```

Comment:
I feel like we can turn off warning for all 404 here. as a client can handle {{TDClientHttpNotFoundException}}

Reply:
It should be OK.